### PR TITLE
Dutch

### PR DESCRIPTION
**Pull request by ltGuillaume**

### CODE REVIEW

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <string name="ongoing_call">Gesprek gaande</string>
     <string name="select_sim">Selecteer de SIM-kaart voor dit gesprek</string>
     <string name="always_use_this_sim">Voor dit nummer altijd deze SIM-kaart gebruiken</string>
-    <string name="unset_default_sim">Unset default SIM</string>
+    <string name="unset_default_sim">Instelling standaard SIM-kaart verwijderen</string>
```
Comment (tibbi) [inline app/src/main/res/values-nl/strings.xml:33]: could this be shortened maybe? It is for a top menu item and it is way too long.

Comment (ltGuillaume) [reply app/src/main/res/values-nl/strings.xml:33]: Yes, but before I change it, is always_use_this_sim also a top menu item?

Comment (tibbi) [reply app/src/main/res/values-nl/strings.xml:33]: no, that is only at the dialog in the middle of the screen, where you can select a SIM card, and a checkbox with the given label is beneath it. Like at file sorting in some apps + use for this folder only.

Comment (ltGuillaume) [reply app/src/main/res/values-nl/strings.xml]: All right, changed 👍

Comment (tibbi) [reply app/src/main/res/values-nl/strings.xml]: ye looks good, thanks

**Pull request merged**